src/replication/log_consumer.hpp

```diff
@@ -153,14 +151,14 @@ namespace cubreplication
 	m_started_tasks--;
       }
 
-      slave_control_channel *get_ctrl_chn ()
+      std::function<void (cubstream::stream_position)> &get_ack_producer ()
```

Comment:
The code is easier to follow if a function wrapper is used instead of getted:
```c
      void ack_produce (cubstream::stream_position sp)
        {
          return m_ack_produce (sp);
        }

  // lc.ack_produce (sp); instead of lc.get_ack_producer () (sp);
```